asn1/CommonLedgerProtocol.asn

```diff
@@ -21,81 +21,53 @@ SideProtocolData ::= SEQUENCE OF SEQUENCE {
   protocolData OCTET STRING
 }
 
-PacketRequest ::= SEQUENCE {
-  packet InterledgerPacket,
+Ack ::= SEQUENCE {
+  ack NULL,
```

Comment:
> make it clear if the request sender requires a response or not
This is defined by the protocol. We shouldn't need a way to infer this from the request.

> avoid the situation where one side sends a response but the other side thinks it's a request, and then responds to it, after which both sides would get confused.

This is why we need a separation between logical request/response and message request/response. A logical response could be async and therefor packaged in a new message request.